.github/workflows/nginx.yml

```diff
@@ -0,0 +1,20 @@
+name: Lint nginx configuration
+
+on:
+  pull_request:
+  push:
+    branches: [main]
+
+jobs:
+  static:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - run: sudo apt-get update
+    - run: sudo apt-get -y install python3-pip python3-setuptools
+    - run: pip3 install wheel
+    - run: pip3 install gixy
+    - run: ~/.local/bin/gixy nginx/nginx.conf
+    - run: ~/.local/bin/gixy nginx/snippets/security-headers.conf
```

Comment:
It doesn't need to separately lint `snippets/security-headers.conf`. It will lint it as part of including it in `nginx.conf` with the line above so you can drop this final line.

Reply:
Removed unnecessary line